taichi/program/kernel.h

```diff
@@ -142,6 +147,10 @@ class TI_DLL_EXPORT Kernel : public Callable {
   bool ir_is_ast_{false};
   // The closure that, if invoked, lauches the backend kernel (shader)
   FunctionType compiled_{nullptr};
```

Comment:
Hmm, are we still using this?

Reply:
@k-ye Yup it's still used in JIT execution  https://github.com/taichi-dev/taichi/blob/master/taichi/program/kernel.cpp#L124 . Will try to get rid of it in a followup PR! (I feel like that PR will be a bit involved so wanted to fix the cgraph bug first ;)